### PR TITLE
Codemotion presentation: the state of tech educatioon

### PR DESCRIPTION
**Pull request by gnunicorn**
Presentation blog post ready for review

### CODE REVIEW

**Comment by gnunicorn** [comment]
@utstikkar, as you are part of the presentation, if you'd do me the honor of reviewing this blog post, that'd be great.

File: _posts/2013-05-26-presenting-at-codemotion-the-state-of-tech-education.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: "Presenting at Codemotion: the state of Tech Education"
+description: ""
+category: 
+tags: ["presentation", "tech education", "berlin"]
+
+teaser: "Actually we wanted to make a co-hosted talk with a few other tech learning inititatives at this years [Codemotion in Berlin](http://berlin.codemotionworld.com/), but when we learnt they were unable to participate, we had to restructure [our talk](http://opentechschool.github.io/slides/presentations/the-state-of-tech-education/) and the topics to talk about. Though a little shy first, we decided to take this opportunity and, after talking about the lastest evolvements in the tech education space and presenting the OpenTechSchool and what we do, talk about something we've been pondering about for a while: **How to make sure learning becomes an on-going process and take a peak into the future plans of the OpenTechSchool**."
```
Comment (utstikkar) [inline _posts/2013-05-26-presenting-at-codemotion-the-state-of-tech-education.md]: it should be "talked about something" (and not "talk")

File: _posts/2013-05-26-presenting-at-codemotion-the-state-of-tech-education.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+image: /assets/content/the-state-of-tech-education.png
+author: ben
+---
+{% include JB/setup %}
+
+![The state of Tech Education](/assets/content/the-state-of-tech-education.png)
+
+Actually we wanted to make a co-hosted talk with a few other tech learning inititatives at this years [Codemotion in Berlin](http://berlin.codemotionworld.com/), but when we learnt they were unable to participate, we had to restructure [our talk](http://opentechschool.github.io/slides/presentations/the-state-of-tech-education/) and the topics to talk about. Though a little shy first, we decided to take this opportunity and, after talking about the lastest evolvements in the tech education space and presenting the OpenTechSchool and what we do, talk about something we've been pondering about for a while: **How to make sure learning becomes an on-going process and take a peak into the future plans of the OpenTechSchool**.
```
Comment (utstikkar) [inline _posts/2013-05-26-presenting-at-codemotion-the-state-of-tech-education.md]: Same as all above comments, obviously.

File: _posts/2013-05-26-presenting-at-codemotion-the-state-of-tech-education.md (hunk 1 above)
Comment (utstikkar) [inline _posts/2013-05-26-presenting-at-codemotion-the-state-of-tech-education.md]: "latest evolvements" does not sounds very good (and might be be the proper way of saying this). I suggest using "lates developments" instead.

Comment (utstikkar) [inline _posts/2013-05-26-presenting-at-codemotion-the-state-of-tech-education.md]: The intro is a bit direct. I think it comes from using "Actually" as the first word. I would rephrase that as "We were initially thinking of sharing/co-hosting a talk with a few ..." or "We initially wanted to share/co-host a talk with a few ...".

**Comment by utstikkar** [comment]
The blog post is good to go once those few points are addressed.
